### DISCUSSION TITLE
Adding a CTF platform.

### DISCUSSION THREAD

**Discussion by Crimson-Typhoon-147** [Vinay-Khanagavi/Cyberbear.github.io#51] [Ideas]
First of all congrats on building this amazing website. The frontend has my heart . The idea is totally fabulous. 

Not a small ideas tho but yaa consider integrating a CTF platform here either by applying any API  of Hack the Box or so or by building it from scratch.

Well i won't say I know how to do that but i'll definitely want to team up with you guys and work on this idea as a contributor.

**Comment by Vinay-Khanagavi** [comment]
Thank you so much for the kind words!

Feel free to join our Discord server for more discussions and collaboration: https://discord.gg/UPHssgCV

**Comment by Crimson-Typhoon-147** [comment]
The discord link got invalid.

**Reply by Crimson-Typhoon-147** [reply]
Here is my discord user name Crimsontyphoon|Cybersec
and here is the id #soumya4985 . You can directly dm me their or can add me in the server.
